public/app/plugins/datasource/cloudwatch/components/QueryHeader.tsx

```diff
@@ -34,12 +35,19 @@ const QueryHeader: React.FC<Props> = ({
   const { queryMode, region } = query;
   const isMonitoringAccount = useIsMonitoringAccount(datasource.resources, query.region);
   const [regions, regionIsLoading] = useRegions(datasource);
+  const emptyLogsExpression = queryMode === 'Logs' && (isCloudWatchLogsQuery(query) ? !query.expression : false);
```

Comment:
Can this check be just `isCloudWatchLogsQuery(query) ? !query.expression : false`? looks like `isCloudWatchLogsQuery` checks for `queryMode === 'Logs'` already.

Reply:
done